python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -919,6 +919,13 @@ def aten〇squeeze〇dim(self: List[int], dim: int) -> List[int]:
 def prim〇NumToTensor〇Scalar(a: float) -> List[int]:
     return []
 
+def prims〇squeeze(a: List[int], dimensions: List[int]) -> List[int]:
```

Comment:
This needs to be upstreamed. Also, can you add tests to it?

Reply:
The upstream ops should already have lots of tests. I think we should just add it upstream.